### ISSUE TITLE
Exports are not being sorted

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#1089]
# Language
Typescript

# Sample input source code
```
export * from './adverse-reasons';
export * from './files';
export * from './underwriters';
export * from './application-notes';
```

# Expected output
```
export * from './adverse-reasons';
export * from './application-notes';
export * from './files';
export * from './underwriters';
```

# Actual output (or error message)
```
export * from './adverse-reasons';
export * from './files';
export * from './underwriters';
export * from './application-notes';
```

**Comment by k2snowman69** [comment]
Created reproduction branch k2snowman69/fixExportSort to help debug the issue.

**Comment by k2snowman69** [comment]
Fixed with 3.2.1
